src/config.ml

```diff
@@ -95,20 +121,24 @@ let merge t (partial : Partial.t) =
   in
   { display     = field t.display     partial.display
   ; concurrency = field t.concurrency partial.concurrency
+  ; terminal_persistence = field t.terminal_persistence partial.terminal_persistence
   }
 
 let default =
   { display     = if inside_dune then Quiet   else Progress
   ; concurrency = if inside_dune then Fixed 1 else Auto
+  ; terminal_persistence = Terminal_persistence.Preserve
```

Comment:
I believe that your new option is the better default actually. What do you think?

Reply:
I think it's a good default, but in lieu of automated tests, I'd prefer to use it for a while before turning it on by default.